egs2/jtubespeech/tts1/local/download.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Takaaki Saeki (The University of Tokyo)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+download_dir=$1
+
+# check arguments
+if [ $# != 1 ]; then
+    echo "Usage: $0 <download_dir>"
+    exit 1
+fi
+
+set -euo pipefail
+
+function gdrive_download () {
+  CONFIRM=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate "https://docs.google.com/uc?export=download&id=$1" -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')
+  wget --load-cookies /tmp/cookies.txt "https://docs.google.com/uc?export=download&confirm=$CONFIRM&id=$1" -O $2
+  rm -rf /tmp/cookies.txt
+}
```

Comment:
You can use `gdown`, which is installed by default in espnet.

Reply:
https://github.com/espnet/espnet/blob/3d83127d074c4af26cc86f30f0773f72a9ad5bbe/egs2/ruslan/tts1/local/data_download.sh#L23